lib/puppet/indirector/indirection.rb

```diff
@@ -60,7 +60,7 @@ def delete
 
   # Set the time-to-live for instances created through this indirection.
   def ttl=(value)
-    raise ArgumentError, "Indirection TTL must be an integer" unless value.is_a?(Integer)
+    raise ArgumentError, _("Indirection TTL must be an integer") unless value.is_a?(Integer)
```

Comment:
Could you add a translator comment that TTL here refers to "time to live", a value that represents time?